packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -57,38 +52,90 @@ const defaultConfigs = {
   },
   USDPERL: {
     type: "medianizer",
-    lookback: 7200,
     invertPrice: true,
     minTimeBetweenUpdates: 60,
     medianizedFeeds: [{ type: "cryptowatch", exchange: "binance", pair: "perlusdt" }]
   },
   BCHNBTC: {
     type: "medianizer",
-    lookback: 7200,
     minTimeBetweenUpdates: 60,
     medianizedFeeds: [
       { type: "cryptowatch", exchange: "coinbase-pro", pair: "BCHBTC" },
       { type: "cryptowatch", exchange: "binance", pair: "BCHBTC" },
       { type: "cryptowatch", exchange: "huobi", pair: "BCHBTC" }
     ]
   },
+  STABLESPREAD: {
+    // This is alternatively known as "STABLESPREAD/ETH"
+    type: "basketspread",
+    lookback: 7200, // We override the default value of 172800, equal to the liquidationLiveness
```

Comment:
See [this comment](https://github.com/UMAprotocol/protocol/pull/2354/files#r550805900) for reasoning why I removed all `lookback` values in this file except for this pricefeed, which depends on a `BalancerPriceFeed` and is used by EMP's with a large lookback window.

Note that the `BCHNBTC` EMP also has a large lookback window of 172800, but it depends only on Cryptowatch Pricefeeds which handle larger lookbacks much better than the DEX-based pricefeeds.